File: back/src/datastructure/response/movie.go
package response

type MovieResponse struct {
	ID          uint     `json:"id"`
	Title       string   `json:"title"`
	SubTitle    string   `json:"subtitle"`
	Description string   `json:"description"`
	ReleaseDate string   `json:"release_date"`
	Genre       string   `json:"genre"`
	Director    string   `json:"director"`
	Cast        []string `json:"cast"`
	Duration    int      `json:"duration"`
	PosterPath  string   `json:"poster_path"`
}

File: front/src/app/types/movie.ts
export interface ShowingInfo {
  screening_id: number
  start_time: string
  end_time: string
}

export interface MovieTLResponse {
  id: number
  title: string
  subtitle: string
  description: string
  release_date: string
  genre: string
  director?: string
  cast: string[]
  duration?: number
  poster_path?: string
}
